server/lib/donations.js

```diff
@@ -32,7 +31,7 @@ export const processDonation = (Sequelize, donation) => {
                   groupId: group.id,
                   groupName: group.name,
                   paymentMethodId: paymentMethod.id,
-                  description: `OpenCollective: ${group.slug}`
+                  description: `https://opencollective.com/${group.slug}`
```

Comment:
We do have that data but I'm not sure what we get out of adding that, since no one ever looks at it. This metadata is meant for the merchant internally to provide info about the payment - groupid, paymentMethodId doesn't mean anything to anyone else. User never sees it on credit card statement. If we were a typical merchant, this would be helpful for manual book keeping. Not clear how it helps us in our current setup?